scripts/beegfspkgs.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+#
+KERNEL=$(uname -r)
+echo "Kernel version is $KERNEL"
+#
+systemctl stop firewalld
+systemctl disable firewalld
+#
+sudo yum -y install epel-release
+yum install -y kernel-devel-${KERNEL} kernel-headers-${KERNEL} kernel-tools-libs-devel-${KERNEL}
+sudo yum -y install gcc gcc-c++
+sudo yum -y install zlib zlib-devel bzip2 bzip2-devel bzip2-libs openssl openssl-devel openssl-libs nfs-utils rpcbind mdadm wget python-pip automake autoconf
+#
+wget -O /etc/yum.repos.d/beegfs-rhel7.repo https://www.beegfs.io/release/beegfs_7_1/dists/beegfs-rhel7.repo
+rpm --import https://www.beegfs.io/release/latest-stable/gpg/RPM-GPG-KEY-beegfs
+#
+echo "net.ipv4.neigh.default.gc_thresh1=1100" | sudo tee -a /etc/sysctl.conf
```

Comment:
I don't think you need to reboot after this script but you will need to run "sysctl -p" for these changes to take effect

Reply:
Removed reboot.